backend/services/mgmt/v1alpha1/integration_tests/user-account-service_integration_test.go

```diff
@@ -554,6 +557,9 @@ func (s *IntegrationTestSuite) Test_UserAccountService_IsAccountStatusValid_Neos
 
 		require.False(s.T(), resp.Msg.GetIsValid())
 		require.NotEmpty(s.T(), resp.Msg.GetReason())
+		require.NotEmpty(s.T(), resp.Msg.GetAccountStatus())
+		require.Equal(s.T(), uint64(101), resp.Msg.GetUsedRecordCount())
```

Comment:
This should just be 50 since that is what the query is returning. The requested count does not change the used record count.

Reply:
beat me to it!